VISUAL_TESTS/readme.md

```diff
@@ -83,6 +83,16 @@ which yields to upper estimates of the temperature  increase due to shear heatin
 | ------------- |-----------------------------|
 | ![](img/ShearHeatingDuretz14.png)  | ![](img/ShearHeatingDuretz14Reference.png) |
 
+### Shrinking
+
+Based on [Yamato et. al., 2022](https://www.sciencedirect.com/science/article/abs/pii/S0012821X2200156X) 
+model and shows deformation and shows alteration of the x component in the inclusion 
```

Comment:
Test from Yamato et al., 2022 (Figure 3) at a resolution of 200 × 200 meshes: 
Figure show the evolution of shear stress for an inclusion submitted to positif density change (compressible mechanical code with linear viscous material fitting the analytical solution) - parameters are in the Table 1 of the paper.
![Fig3_100steps_Yamatoetal2022](https://user-images.githubusercontent.com/47114031/215820208-c0d713e2-e5a3-4beb-be69-739b96c8899d.gif)